bvbabel/vmr.py

```diff
@@ -260,8 +267,12 @@ def write_vmr(filename, header, data_img):
         # VMR Pre-Data Header
         # ---------------------------------------------------------------------
         # Expected binary data: unsigned short int (2 bytes)
-        data = header["File version"]
-        f.write(struct.pack('<H', data))
+        if filename[-3:] == 'vmr':   # check file format used
+            data = header["File version"]
+            f.write(struct.pack('<H', data))
+            fformat = '<B'
```

Comment:
Hmm, I also don't like the extra `fformat` variable here. This goes back to may top comment of having a separate function to read non-vmr files.

Reply:
Other commits in this PR looks are fine. I am not sure about this glued-on v16 read/write.